policyengine_us/parameters/gov/states/co/ccap/base_parent_fee/first_and_second_multiplication_factor.yaml

```diff
@@ -1,12 +1,21 @@
-description: Colorado Child Care Assistance Program multiplies adjusted gross income or federal poverty guidelines with this factor to calculate base parent fee.
+description: Colorado Child Care Assistance Program multiplies adjusted gross income by the following factor to calculate base parent fee, based on gross income relative to the federal poverty guidelines.  
 metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Colorado Child Care Assistance Program multiplication factor
   reference:
     # This factor is included in the calculation of base parent fee in the spreadsheets
     - title: At 100% FPG Parent Fee Tool
       href: https://docs.google.com/spreadsheets/d/1EEc3z8Iwu_KRTlBtd2NssDDEx_FITqVq/edit#gid=468321263
```

Comment:
While you're here could you change the reference to publicly available documents?